helm.go

```diff
@@ -82,16 +82,23 @@ func generate(log Logger, si ServiceItem, basePath, outputDir string) error {
 	return nil
 }
 
-func helmInstall(log Logger, si ServiceItem, image, tag, outputDir string) (string, error) {
+func helmInstall(log Logger, si ServiceItem, service Service, outputDir string, _ bool) (string, error) {
 	log.Printf("Running helm install for service: %v", si.Name)
 	chartPath := path.Join(outputDir, si.Name)
 	cmd := []string{
-		"helm", "install", si.Name, chartPath,
+		"helm", "upgrade", si.Name, chartPath,
```

Comment:
since `upgrade --install` is working instead of `install`, we dont need upgradeFlag anymore?